rules.csv

```diff
@@ -630,7 +630,7 @@ c,MISRA-C-2012,RULE-2-7,Yes,Advisory,,,There should be no unused parameters in f
 c,MISRA-C-2012,RULE-3-1,Yes,Required,,,The character sequences /* and // shall not be used within a comment,M2-7-1,Syntax,Easy,
 c,MISRA-C-2012,RULE-3-2,Yes,Required,,,Line-splicing shall not be used in // comments,,Syntax,Easy,
 c,MISRA-C-2012,RULE-4-1,Yes,Required,,,Octal and hexadecimal escape sequences shall be terminated,A2-13-1 M2-13-2,Syntax,Medium,
-c,MISRA-C-2012,RULE-4-2,No,Advisory,,,Trigraphs should not be used,A2-5-1,,Import,
```

Comment:
are you sure it was unintentional?

[the commit where it was changed](https://github.com/github/codeql-coding-standards/commit/274eb1e7e9702e72012e7c23cdacd61759398cc2#diff-cdb9077faf1b88ec6d03150af6359fa4acc5d204ad16528b20342d5a3f93fc66L633) looks purposeful - as the Banned pkg label was also rm'd

[the A2-5-1 rule is marked](https://github.com/github/codeql-coding-standards/blob/main/rules.csv#L140) as non queryable and the 2 rule descriptions seem to be truly import to me

maybe this just needs the extra "Trigraphs/digraphs are implemented by the preprocessor in a such a way that we may not have that information in our database. We could look for compiler flags here."  that the A2-5-1 rule has, or maybe we need to re-verify that this info is not in the DB?

Reply:
You are absolutely right - good catch. I was confused by the fact it had an `Import` level, which I would only expect for rules which are implementable. However, that is just a consequence of this being first marked as queryable then moved to not queryable.